logstash-output-amazon_es.gemspec

```diff
@@ -21,14 +21,15 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency 'concurrent-ruby'
-  s.add_runtime_dependency 'elasticsearch', ['>= 1.0.10']
+  s.add_runtime_dependency 'elasticsearch', '~> 1.0', '>= 1.0.10'
   s.add_runtime_dependency 'stud', ['>= 0.0.17', '~> 0.0']
   s.add_runtime_dependency 'cabin', ['~> 0.6']
-  s.add_runtime_dependency "logstash-core", ">= 1.4.0", "< 3.0.0"
-  s.add_runtime_dependency "aws-sdk", ['>= 2.1.14', '~> 2']
+  s.add_runtime_dependency 'logstash-core-plugin-api', '>= 1.60', '<= 2.99'
+  s.add_runtime_dependency 'aws-sdk', '>= 2.1.14', '< 2.4'
```

Comment:
Ahh... nevermind. It looks like we need to add some fields to CredentialConfig to reflect additional options used by CredentialProviderChain in newer versions of the SDK.

I'll add those fields.